pkg/beaconfqdn/writer.go

```diff
@@ -63,21 +63,6 @@ func (w *writer) start() {
 					}).Error(err)
 				}
 
-				// update hosts table with icert updates
-				if data.hostIcert.query != nil {
```

Comment:
The invalid certs are currently being marked in the beacons module and gets used for scoring. If we add it to the other beacons modules, there is a potential for it to be redundantly counted and to skew the scoring. There is also the potential that it might not be counted at all if we leave it out of this module. With this in mind, invalid certs should probably be it's own scoring metric outside of any modules. We can create an issue for that and discuss it later if that sounds cool?